reV/config/execution.py

```diff
@@ -206,5 +206,5 @@ def walltime(self):
         _hpc_walltime : int
             Requested single node job time in hours.
         """
-        self._hpc_walltime = int(self.get('walltime', self._hpc_walltime))
+        self._hpc_walltime = float(self.get('walltime', self._hpc_walltime))
```

Comment:
is this what was causing jobs to die if you request < 1hr?

Reply:
Yep! 0.5 hours got rounded down to 0.0 which slurm didnt like. Annoying little bug!